spec/models/video_spec.rb

```diff
@@ -2,98 +2,56 @@
 
 
 describe Video do
-  it { should belong_to :category }
-  it { should validate_presence_of :title }
-  it { should validate_presence_of :description }
+  it { is_expected.to belong_to :category }
+  it { is_expected.to validate_presence_of :title }
+  it { is_expected.to validate_presence_of :description }
+  it { is_expected.to have_many :reviews }
+
+  describe ".search_by_title" do
+    let!(:videos) do
+      videos = 5.times.map { |i| Fabricate(:video, title: "Lost #{i+1}") }
```

Comment:
Thanks @tomtomecek !
I tried `sequence` actually, but it made some bugs in these suites, because I need exactly `"Lost 1", "Lost 2", ..."Lost 5"`. But `sequence` will remember the previously created numbers, it increments after each test case. So sometimes I got `"Lost 6",...."Lost 10"`, or `"Lost 15",..."Lost 20"`